applications/DEMApplication/custom_elements/spheric_particle.h

```diff
@@ -105,7 +105,7 @@ array_1d<double, 3> mDomainMin;
 array_1d<double, 3> mDomainMax;
 SphericParticle* mpThisParticle;
 SphericParticle* mpOtherParticle;
-Node<3>* mpOtherParticleNode;
+Node* mpOtherParticleNode;
```

Comment:
Not related, but this is dangerous, we should take a look....

Reply:
@KratosMultiphysics/dem

Reply:
I see no real problem. It’s part of a memory buffer, so it must be fast and parallel and avoid copies of smart pointers.